main.js

```diff
@@ -22,3 +22,148 @@ spreadList.forEach((list) => {
+  description: 'Just a dummy webpage initially created to be my portfolio, created with only HTML and javascript with no template guide. Zero functionality. Just a dummy webpage, on interractive.',
+  technology: ['Bootstrap', 'Html', 'css'],
+  sourceCode: '',
+  liveDemo: '',
+}, {
+  projectTitle2: 'Multi Post Stories',
+  projectImage: 'secondpage.svg',
+  projectTitle: 'Professional works',
+  demo: 'See Project',
+  source: 'See Source',
+  description2: 'Just a dummy webpage initially created to be my portfolio, created with only HTML and javascript with no template guide. Zero functionality. Just a dummy webpage, on interractive.',
+  description: 'Just a dummy webpage initially created to be my portfolio, created with only HTML and javascript with no template guide. Zero functionality. Just a dummy webpage, on interractive.',
+  technology: ['Bootstrap', 'Html', 'css'],
+  sourceCode: '',
+  liveDemo: '',
+}, {
+  projectTitle2: 'Multi Post Stories',
+  projectImage: 'image.jpeg',
+  projectTitle: 'Professional works',
+  demo: 'See Project',
+  source: 'See Source',
+  description2: 'Just a dummy webpage initially created to be my portfolio, created with only HTML and javascript with no template guide. Zero functionality. Just a dummy webpage, on interractive.',
+  description: 'Just a dummy webpage initially created to be my portfolio, created with only HTML and javascript with no template guide. Zero functionality. Just a dummy webpage, on interractive.',
+  technology: ['Bootstrap', 'Html', 'css'],
+  sourceCode: '',
+  liveDemo: '',
+}, {
+  projectTitle2: 'Multi Post Stories',
+  projectImage: 'secondpage.svg',
+  projectTitle: 'Professional works',
+  demo: 'See Project',
+  source: 'See Source',
+  description2: 'Just a dummy webpage initially created to be my portfolio, created with only HTML and javascript with no template guide. Zero functionality. Just a dummy webpage, on interractive.',
+  description: 'Just a dummy webpage initially created to be my portfolio, created with only HTML and javascript with no template guide. Zero functionality. Just a dummy webpage, on interractive.',
+  technology: ['Bootstrap', 'Html', 'css'],
+  sourceCode: '',
+  liveDemo: '',
+}, {
+  projectTitle2: 'Multi Post Stories',
+  projectImage: 'image1.jpeg',
+  projectTitle: 'Professional works',
+  demo: 'See Project',
+  source: 'See Source',
+  description2: 'Just a dummy webpage initially created to be my portfolio, created with only HTML and javascript with no template guide. Zero functionality. Just a dummy webpage, on interractive.',
+  description: 'Just a dummy webpage initially created to be my portfolio, created with only HTML and javascript with no template guide. Zero functionality. Just a dummy webpage, on interractive.',
+  technology: ['Bootstrap', 'Html', 'CSS'],
+  sourceCode: '',
+  liveDemo: '',
+},
+];
+
+const modalSection = document.querySelector('.prof');
+const card = document.querySelector('.biew');
```

Comment:
- [ ] Your modal looks great, but on some mobile screens, it does not contain the entire content leaving the buttons to be displayed out of the content like on the Iphone SE. Please make sure your modal is responsive and it displays correctly on every screen size.
 -Iphone SE view:
<img width="475" alt="image" src="https://user-images.githubusercontent.com/61919617/189110640-e3a6f4d5-dd9b-47f1-aa4c-eafcba2a06d3.png">

Reply:
Thank you for pointing this out. I am already working on it

Reply:
Thanks, I have already worked on it and also fixed the view on 360px screen sizes like Samsung mobile screens